recipes/earcut/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class EarcutConan(ConanFile):
+    name = "earcut"
+    description = "A C++ port of earcut.js, a fast, header-only polygon triangulation library."
+    homepage = "https://github.com/mapbox/earcut.hpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "ISC"
+    topics = ("algorithm", "cpp", "geometry", "rendering", "triangulation",
+              "polygon", "header-only", "tessellation", "earcut")
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _minimum_compilers_version(self):
+        # References:
+        # * https://github.com/mapbox/earcut.hpp#dependencies
+        # * https://en.cppreference.com/w/cpp/compiler_support/11
+        # * https://en.wikipedia.org/wiki/Xcode#Toolchain_versions
+        return {
+            "apple-clang": "5.1",
+            "clang": "3.4",
+            "gcc": "4.9",
```

Comment:
```suggestion
            "gcc": "4.9",
```

I would say 4.7 is enough for most C++11 features, I used it for legacy projects with C++11. But I'm fine with 4.9 due its age.

Reply:
This particular requirement was taken from the library's [readme](https://github.com/mapbox/earcut.hpp#dependencies).